python/tank/context.py

```diff
@@ -1226,7 +1208,7 @@ def from_entity(tk, entity_type, entity_id):
     return _from_entity_type_and_id(tk, dict(type=entity_type, id=entity_id))
 
 
-def _from_entity_type_and_id(tk, entity, source_entity=None, additional_entities=None, user=None):
```

Comment:
This change was caused by the revert of my original pickle fix that was merged some time ago in the Python 3 branch.